webpack.config.extensions.js

```diff
@@ -114,12 +114,6 @@ module.exports = [
 	},
 	{
 		...componentsWebpackConfig,
-		output: {
```

Comment:
Fixed by https://github.com/Automattic/wp-calypso/pull/34860? @ockham

Reply:
Yes! Good spot. Sorry for not cleaning up after myself :sweat_smile: